src/meta/app/src/principal/user_stage.rs

```diff
@@ -402,6 +402,7 @@ pub struct StageParams {
     pub storage: StorageParams,
 }
 
+// todo(yangxiufeng): not used, remove it later.
```

Comment:
Remove in another PR or this PR?

Reply:
In another pr,  used in many compat tests about meta